nautobot/extras/models/jobs.py

```diff
@@ -653,16 +652,19 @@ def enqueue_job(
         if celery_kwargs is not None:
             job_celery_kwargs.update(celery_kwargs)
 
+        # When running synchronously (explicitly on this call or globally), the celery worker
+        # setup_nautobot_job_logging() doesn't get called by a worker signal, so we need to do it here.
         if synchronous or getattr(settings, "CELERY_TASK_ALWAYS_EAGER", False):
+            setup_nautobot_job_logging(None, None, app.conf)
```

Comment:
My only concern with this instead of calling it in the test runner setup is this runs for every call of `enqueue_job` during testing

Reply:
It's not an expensive process but it is something that we should consider when thinking about optimizing our test run time

Reply:
That's a thought... I can see whether we can move it there.